R/bbr.R

```diff
@@ -321,9 +321,22 @@ bbi_help <- function(.cmd_args=NULL) {
 #'   model. `submit_model()` and `submit_models()` also support passing a
 #'   configuration file via the `.config_path` argument.
 #'
+#'  **Notes on `.nonmem_dir`**
+#'
+#'   `bbi` tries to detect subdirectories for different NONMEM versions, along
+#'   with a license file and executable within the subdirectory. `.nonmem_dir`
+#'   should therefore point to the *parent* directory containing the executable.
+#'
+#'   If your NONMEM executable exists in the the location `/opt/NONMEM/nm75/run`,
```

Comment:
"exists in the location `/opt/NONMEM/nm75/run`"  here must mean under the `run/` directory, but that could be interpreted as though run itself is the executable file of interest.

I wonder if this is the kind of thing were an example layout would help a lot.  For example

```
**Notes on `.nonmem_dir`**

`.nonmem_dir` should point to the parent directory that contains one
or more NONMEM installations.  It should not point to the installation
directory itself.  Here is an example:

    /opt/NONMEM/
    |-- nm74
    |   |-- license
    |   |-- run
    |   |-- source
    |   | ...
    |   `-- util
    `-- nm75
        |-- license
        |-- run
        |-- source
        | ...
        `-- util

`/opt/NONMEM/` contains two NONMEM installations, one for NONMEM 7.5
and one for NONMEM 7.5.  In this case, you should pass "/opt/NONMEM"
as `.nonmem_dir` and either "nm74" or "nm75" as `.nonmem_version`.
```

Reply:
I left the first section, but removed the second mention of the executable in favor of your example layout